### PR TITLE
[term entry]- kotlin offsetAt

### LINKED ISSUE #3815: [Term Entry] Kotlin Datetime .offsetAt()
### Reason/inspiration (optional)

We would like a new term entry on the `.offsetAt()` method for Kotlin: [Datetime](https://www.codecademy.com/resources/docs/kotlin/datetime). The entry should go in a new file under `docs/content/kotlin/concepts/datetime/terms/offsetAt/offsetAt.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```kotlin `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by vrun1208**
### Description

closes #3815 
Added a new term entry for kotlin dateTime .offsetAt() method

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by Atharva1723 — APPROVED** [review]
LGTM 👍

**Comment by vrun1208** [comment]
Hi @ishg-153, how can I be the maintainer of this repo?

**Review by vaibhavmani810 — COMMENTED** [review]
Hi @vrun1208 Looks Fine! , Sending it for second review.

File: content/kotlin/concepts/datetime/terms/offsetAt/offsetAt.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+The **`.offsetAt()`** method is used to find the offset from UTC(Coordinated Universal time) for a specific time zone at a specified `instant` in physical time.
+
+## Syntax
+
+```pseudo
+fun TimeZone.offsetAt(instant: Instant): UtcOffset
+```
+
+- `Instant`: A moment in time.
+- `utcOffset`: An offset from UTC (Coordinated Universal Time).
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/offsetAt/offsetAt.md:24]:
```suggestion
- `UtcOffset`: An offset from UTC (Coordinated Universal Time).
```

File: content/kotlin/concepts/datetime/terms/offsetAt/offsetAt.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  val offset = timeZone.offsetAt(instant)
+
+  println("At $instant, the offset from UTC in ${timeZone.id} is $offset.")
+}
+
+```
+
+> **Note:** The string `2023-11-07T12:00:00Z` specifies date and time of November 7, 2023, at 12:00:00 UTC. The `T` separates the date and time components, and `Z` indicates that the time is in UTC.
+
+Output:
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/offsetAt/offsetAt.md:48]:
```suggestion
The above example will result in the following output:
```

File: content/kotlin/concepts/datetime/terms/offsetAt/offsetAt.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+```shell
+At 2023-11-07T12:00:00Z, the offset from UTC in Asia/Kolkata is +05:30.
+
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/datetime/terms/offsetAt/offsetAt.md:52]:
```suggestion
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Here are some requested changes from my side.

**Comment by vrun1208** [comment]
> Here are some requested changes from my side.

@dakshdeepHERE, I commit the changes as you suggested!

**Review by dakshdeepHERE — APPROVED** [review]
Looks good to merge

**Pull request merged**